tests_integration/nativeapp/test_validate.py

```diff
@@ -13,20 +13,63 @@
 # limitations under the License.
 
 from shlex import split
+from textwrap import dedent
+
+from tests.nativeapp.factories import (
+    ProjectV2Factory,
+    ApplicationPackageEntityModelFactory,
+    ApplicationEntityModelFactory,
+    ProjectV10Factory,
+)
 from tests.project.fixtures import *
 
 
+@pytest.mark.integration
+def test_nativeapp_validate_v1(nativeapp_project_directory, runner, temp_dir):
+    ProjectV10Factory(
+        pdf__native_app__name="myapp",
+        pdf__native_app__artifacts=[
+            {"src": "app/*", "dest": "./"},
+        ],
+        files={
+            "app/setup.sql": "CREATE OR ALTER VERSIONED SCHEMA core;",
+            "app/README.md": "\n",
+            "app/manifest.yml": "\n",
+        },
+    )
+    with nativeapp_project_directory(temp_dir):
```

Comment:
```suggestion
    with nativeapp_teardown(temp_dir):
```
it's not necessary to use `nativeapp_project_directory` since we're not using an example project